apps/src/hamburger/hamburger.js

```diff
@@ -9,6 +9,8 @@ export const initHamburger = function() {
   $(function() {
     $('#hamburger-icon').click(function(e) {
       $(this).toggleClass('active');
+      $('#hamburger').removeClass('user-is-tabbing');
+      $('#help-button').removeClass('user-is-tabbing');
```

Comment:
Is there a reason we needed to manage every button's state rather than the `<body>` tag for the whole page like the article you linked does? I'm just thinking about the future where we extend this to more buttons/ui elements. This list will keep growing.

Reply:
Yep, I tried adding the class to the `header_right` element, but inheriting the outline property doesn't result in the same behavior - it seems to only evaluate once at the beginning, so outlines are always turned off. You're right, though, that we should restructure this solution in the future to be more extensible.